lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -74,24 +78,46 @@ export default function LMSFilePicker({
         authToken,
         path: listFilesApi.path,
       }));
-      setDialogState({ ...INITIAL_DIALOG_STATE, state: 'fetched', files });
+      const title =
+        files.length === 0 ? 'No files available' : INITIAL_DIALOG_STATE.title;
+      const buttonAction =
+        files.length === 0 ? 'none' : INITIAL_DIALOG_STATE.buttonAction;
```

Comment:
Maybe can the condition (0 files) could be check one on a single if statement. Alternative it can be done inside the setDialogState avoiding variables)